### PR TITLE
add clickable function to allow to go back

### PR DESCRIPTION
**Pull request by mytechtip**
Currently only going back. Password wont be retrieved as it is not viewable anyway.

### CODE REVIEW

File: src/ts/components/account/Progress.tsx [hunk 1]
```diff
@@ -11,6 +15,17 @@ interface IProgressProps {
  */
 class Progress extends React.Component<IProgressProps> {
 
+  handleClick = route => event => {
```
Comment (fyang1024) [inline src/ts/components/account/Progress.tsx]: unused event generates a tslint warning here. perhaps we can replace it with `()`

File: src/ts/components/account/Progress.tsx [hunk 2]
```diff
@@ -11,6 +15,17 @@ interface IProgressProps {
...
+    // only allow to go back
+    if (this.props.progress > 1 && route === CREATE_PASSWORD_ROUTE) {
```
Comment (ttaoS) [inline src/ts/components/account/Progress.tsx]: so progress 1 => create pwd, 2 => generate phrase, 3 => confirm phrase? for some reason, `confirm phrase` seems not change progress icon.

Comment (ttaoS) [reply src/ts/components/account/Progress.tsx]: image

Comment (mytechtip) [reply src/ts/components/account/Progress.tsx]: According to the screen shot, confirm screen is still under progress step 2.

File: src/ts/components/account/Progress.tsx (hunk 1 above)
Comment (mytechtip) [reply src/ts/components/account/Progress.tsx]: yeah, you are right

File: src/ts/components/account/Progress.tsx (hunk 2 above)
Comment (ttaoS) [reply src/ts/components/account/Progress.tsx]: So the new account created page will be step 3?

Comment (mytechtip) [reply src/ts/components/account/Progress.tsx]: there is another step (step 3) to select chain according to the screenshot

Comment (fyang1024) [reply src/ts/components/account/Progress.tsx]: I found this is a bit confusing as well, but then I checked zeplin mockup and found it was like this

Comment (ttaoS) [reply src/ts/components/account/Progress.tsx]: image
yeah. no one actually on step 3 :)

Comment (fyang1024) [reply src/ts/components/account/Progress.tsx]: Screen Shot 2019-04-22 at 1 43 55 pm

Comment (fyang1024) [reply src/ts/components/account/Progress.tsx]: Screen Shot 2019-04-22 at 11 39 59 pm

Comment (ttaoS) [reply src/ts/components/account/Progress.tsx]: i see. I think we need to revisit this later.

**Pull request merged**